src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandlerTest.java

```diff
@@ -1,76 +1,246 @@
+        // Then
+        assertThat(response.getErrors()).contains(expectedErrorMessage);
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "witnessOption1UploadDate,documentUploadWitness1,Invalid date: witness statement date entered must not be in the future (1).",
+        "witnessOption3UploadDate,documentUploadWitness3,Invalid date: witness statement date entered must not be in the future (2)."
+    })
+    void shouldReturnError_whenWitnessOptionUploadDateInFuture(String dateField, String collectionField, String expectedErrorMessage) {
+        // Given
+        List<Element<UploadEvidenceWitness>> date = newArrayList();
+        date.add(0, element(invoke(uploadEvidenceDate2.toBuilder(), dateField, time.now().toLocalDate().plusWeeks(1)).build()));
+
+        CaseData caseData = invoke(CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder(), collectionField, date)
+            .build();
+        CallbackParams params = callbackParamsOf(caseData, MID, PAGE_ID);
+
+        // When
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        // Then
+        assertThat(response.getErrors()).contains(expectedErrorMessage);
     }
 
     @Test
-    void givenSubmittedThenReturnsSubmittedCallbackResponse() {
-        CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build();
-        CallbackParams params = callbackParamsOf(caseData, CallbackType.SUBMITTED);
+    void shouldReturnError_whenOneDateIsInFuture() {
+        //documentUploadWitness1 represents a collection so can have multiple dates entered at any time,
+        // these dates should all be in the past, otherwise an error will be populated
+
+        // Given
+        List<Element<UploadEvidenceWitness>> date = newArrayList();
+        date.add(0, element(uploadEvidenceDate2.toBuilder()
+                                .witnessOption1UploadDate(time.now().toLocalDate().minusWeeks(1)).build()));
+        date.add(1, element(uploadEvidenceDate2.toBuilder()
+                                .witnessOption1UploadDate(time.now().toLocalDate().minusWeeks(1)).build()));
+        //dates above represent valid past dates, date below represents invalid future date.
+        date.add(2, element(uploadEvidenceDate2.toBuilder()
+                                .witnessOption1UploadDate(time.now().toLocalDate().plusWeeks(1)).build()));
+
+        CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder()
+            .documentUploadWitness1(date)
+            .build();
+        CallbackParams params = callbackParamsOf(caseData, MID, PAGE_ID);
+
+        // When
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        // Then
+        assertThat(response.getErrors()).contains("Invalid date: witness statement date entered must "
+                                                      + "not be in the future (1).");
+    }
```

Comment:
Using the approach used in `shouldReturnError_whenExpertOptionUploadDateFuture`, make this test run against any of the dates.

As I appreciate there's a bit of ambiguity here, give me a shout before doing it so I can explain better what I have in mind.